feat(content): Remove interim spaceport visits during FW Escort (#10747)
endless-sky/endless-sky: data/human/free worlds 0 prologue.txt

> # Deprecated in v0.10.11. This event is preserved in case a now deleted mission calling this event is already active.

Reviewer: ```suggestion
# Deprecated in v0.10.11. This event is preserved in case an older version of the mission "FW Escort 1B" is already active upon updating.
```

	to fail

> 		or
> 			has "chosen sides"

Reviewer: ```suggestion
		has "chosen sides"
```

mission "FW Escort 2"
	autosave
	landing
	name "Free Worlds Escort"
	description "Escort a Free Worlds supply convoy safely to <destination>."
	source "Longjump"
	destination "Wayfarer"
	to offer
		has "FW Escort 1B: done"
	to fail
		has "chosen sides"
	
	on offer
		conversation
			`Tomek is waiting in <origin>'s spaceport to receive the freighters. After double-checking that the cargo is all intact, he turns to you and the rest of the escort captains and says, "All of you have done a great service for the Free Worlds." He walks over and personally presents a credit chip to each captain.`

> 			`	After handing over your payment, Tomek announces, "The convoy is heading back out for <planet> immediately. Same deal as before: 400,000 credits for the safe return of the freighters, their crew, and their cargo. And I swear your courage will never be forgotten by the Council or the Free Worlds. If you wish to volunteer a second time, please stay here."`

Reviewer 2: "If you wish" sounds like volunteering is personally beneficial to you.

Also, "back out" is a bit weird since you're heading from one industrial centre to another, right?

```suggestion
			`	After handing over your payment, Tomek announces, "The convoy is heading back to <planet> immediately. Same deal as before: 400,000 credits for the safe return of the freighters, their crew, and their cargo. And I swear your courage will never be forgotten by the Council or the Free Worlds. If you're willing to volunteer a second time, please stay here."`
```

			`	Tomek leaves, presumably to gather replacements for the captains who have already run off.`
			choice
				`	(Volunteer.)`
				`	(Ditch the escort group.)`
					decline
			

> 			`	You stick around with a few of the other escort pilots, and Tomek returns, luckily with some new faces in tow. "You know the drill," he says. "We can't afford to lose any of these freighters. Good luck to all of you, and I pray to see you all safely back here in a couple of weeks."`

Reviewer 2: ```suggestion
			`	You stick around with a few of the other escort pilots and watch as most of the others wander away. Soon Tomek returns, luckily with some new faces in tow. "You know the drill," he says. "We can't afford to lose any of these freighters. Good luck to all of you, and I pray to see you all safely back here in a couple of weeks."`
```

mission "FW Escort 2"
	autosave
	landing
	name "Free Worlds Escort"
	description "Escort a Free Worlds supply convoy safely to <destination>."
	source "Longjump"
	destination "Wayfarer"
	to offer
		has "FW Escort 1B: done"
	to fail
		has "chosen sides"
	
	on offer
		conversation

> 			`Tomek is waiting on <origin>'s spaceport to receive the freighters. After double-checking that the cargo is all intact, he turns to you and the rest of the escort captains and says, "All of you have done a great service for the Free Worlds." He walks over and personally presents a credit chip to each captain.`

Reviewer 3: ```suggestion
			`Tomek is waiting in <origin>'s spaceport to receive the freighters. After double-checking that the cargo is all intact, he turns to you and the rest of the escort captains and says, "All of you have done a great service for the Free Worlds." He walks over and personally presents a credit chip to each captain.`
```
